roles/sap_ha_cluster/tasks/configure_srhook.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: "{{ __tname }} Create srHook shared directory"
+  ansible.builtin.file:
+    path: "{{ sap_ha_cluster_hadr_provider_path }}"
+    state: directory
+    mode: "0755"
+    owner: "{{ sap_ha_cluster_hana_sid | lower }}adm"
+    group: sapsys
+
+- name: "{{ __tname }} Copy srHook to shared directory"
+  ansible.builtin.copy:
+    remote_src: true
+    src: /usr/share/SAPHanaSR/srHook/SAPHanaSR.py
+    dest: "{{ sap_ha_cluster_hadr_provider_path }}/{{ sap_ha_cluster_hadr_provider_name }}.py"
+    mode: "0755"
+    owner: "{{ sap_ha_cluster_hana_sid | lower }}adm"
+    group: sapsys
+  # Do not run in check mode because the path is created in the previous step
+  when: not ansible_check_mode
+
+- name: "{{ __tname }} Check global.ini for 'ha_dr_saphanasr'"
+  ansible.builtin.shell: |
+    grep ha_dr_saphanasr /usr/sap/{{ sap_ha_cluster_hana_sid | upper }}/SYS/global/hdb/custom/config/global.ini
+  register: __sap_ha_cluster_srhook_trace_global
```

Comment:
Please include the "register" string in all variables which are registered, as per https://sap-linuxlab.github.io/contribute/contribute_04_dev_guidelines/.